webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const webpack = require("webpack");
```

Comment:
move the webpack and postcss logic to client as well. Its needed only for report gen.